utils/p4runtime_lib/convert.py

```diff
@@ -128,8 +139,15 @@ def encode(x, bitwidth):
     assert(not matchesIPv4('1000.0.0.1'))
     assert(not matchesIPv4('10001'))
 
+    assert(matchesIPv6('2001:0db8:85a3:0000:0000:8a2e:0370:7334'))
+    assert(not matchesIPv6('2001:0db8:85a3:0000:0000:8a2e:0370'))
+    assert(not matchesIPv6('2001:0db8:85a3:0000:0000:8a2e::7334'))
+    assert(not matchesIPv6('2001:0db8:85a3::8a2e:0370:7334'))
```

Comment:
Same comment for this one.

Reply:
Have you run these tests?

Reply:
- Yes, I have tested `2001:0db8:85a3:0000:0000:8a2e:0370` and it gives `False` apparently. The other two I have changed, which were added by mistake.

- Tested cases can be found [here](https://onecompiler.com/python/42ak4r6e5)